<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vastMainWindow</class>
 <widget class="QMainWindow" name="vastMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vast</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>771</width>
         <height>481</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="topScrollAreaGridLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="centralGridLayout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="bottomScrollAreaGridLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="bottomGridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="topGridLayout" rowstretch="0">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="leftGridLayout"/>
    </item>
    <item row="1" column="2">
     <layout class="QGridLayout" name="rightGridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="separator"/>
    <addaction name="action_Save_Image"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionSave_all_Images"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Preferences">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuShow">
     <property name="title">
      <string>Show...</string>
     </property>
     <addaction name="actionShow_Labels"/>
     <addaction name="actionShow_Crosshair"/>
    </widget>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="menuShow"/>
    <addaction name="actionIgnore_Orientation"/>
    <addaction name="actionPropagate_Zooming"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Zmap_Mode"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionFind_Global_Min"/>
    <addaction name="actionFind_Global_Max"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionKey_Commands"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Preferences"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_image">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/openimage.png</normaloff>:/common/openimage.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Image(s)...</string>
   </property>
   <property name="shortcut">
    <string>O, I</string>
   </property>
  </action>
  <action name="action_Save_Image">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/saveimage.gif</normaloff>:/common/saveimage.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Image</string>
   </property>
   <property name="shortcut">
    <string>S, I</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/saveimageas.gif</normaloff>:/common/saveimageas.gif</iconset>
   </property>
   <property name="text">
    <string>Save Image As...</string>
   </property>
   <property name="toolTip">
    <string>Save Image As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/exit.png</normaloff>:/common/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/preferences_settings.png</normaloff>:/common/preferences_settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>S, P</string>
   </property>
  </action>
  <action name="actionFind_Global_Min">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/min.gif</normaloff>:/common/min.gif</iconset>
   </property>
   <property name="text">
    <string>Find Local Min</string>
   </property>
   <property name="toolTip">
    <string>Find Local Min</string>
   </property>
   <property name="shortcut">
    <string>F, M, I</string>
   </property>
  </action>
  <action name="actionFind_Global_Max">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/max.gif</normaloff>:/common/max.gif</iconset>
   </property>
   <property name="text">
    <string>Find Local Max</string>
   </property>
   <property name="toolTip">
    <string>Find Local Max</string>
   </property>
   <property name="shortcut">
    <string>F, M, A</string>
   </property>
  </action>
  <action name="actionShow_Labels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/orient.gif</normaloff>:/common/orient.gif</iconset>
   </property>
   <property name="text">
    <string>Labels</string>
   </property>
   <property name="shortcut">
    <string>S, L</string>
   </property>
  </action>
  <action name="actionShow_scaling_option">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/autoscale.gif</normaloff>:/common/autoscale.gif</iconset>
   </property>
   <property name="text">
    <string>Show Scaling Options</string>
   </property>
   <property name="toolTip">
    <string>Show Scaling Options</string>
   </property>
   <property name="shortcut">
    <string>S, S, O</string>
   </property>
  </action>
  <action name="actionIgnore_Orientation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore Orientation</string>
   </property>
  </action>
  <action name="actionPropagate_Zooming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Propagate Zooming</string>
   </property>
  </action>
  <action name="actionShow_Crosshair">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Crosshair</string>
   </property>
   <property name="shortcut">
    <string>S, C</string>
   </property>
  </action>
  <action name="actionCenter_to_ca">
   <property name="text">
    <string>Center To Ca</string>
   </property>
   <property name="toolTip">
    <string>Center To Ca</string>
   </property>
   <property name="shortcut">
    <string>Space, Space</string>
   </property>
  </action>
  <action name="actionSave_all_Images">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/saveAll.gif</normaloff>:/common/saveAll.gif</iconset>
   </property>
   <property name="text">
    <string>Save All Images</string>
   </property>
   <property name="toolTip">
    <string>Save All Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+S</string>
   </property>
  </action>
  <action name="actionToggle_Zmap_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ZMap Mode</string>
   </property>
   <property name="toolTip">
    <string>ZMap Mode</string>
   </property>
   <property name="shortcut">
    <string>T, Z, M</string>
   </property>
  </action>
  <action name="actionKey_Commands">
   <property name="text">
    <string>Key Commands</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/viewer.qrc"/>
 </resources>
 <connections/>
</ui>
